scripts/ast-patches/utils.ts

```diff
@@ -0,0 +1,19 @@
+import ts from "typescript";
+import Constants from "../constants";
+export type Writable<T> = { -readonly [P in keyof T]: T[P] };
+
+export const is$wModule = (statement: ts.Statement): boolean =>
```

Comment:
If you change this to use `is` you won't have to write all the `as ts.ModuleDeclaration` statements like:
`  const $w = ast.statements.find(is$wModule) as ts.ModuleDeclaration;`
It's also safer since find can actually return undefined.
```suggestion
export const is$wModule = (statement: ts.Statement): statement is ts.ModuleDeclaration =>
```